app/src/main/res/layout/fragment_main_page.xml

```diff
@@ -16,67 +17,269 @@
-                android:text="@string/action_see_more"
-                android:textColor="@android:color/holo_blue_dark"
-                android:textSize="@dimen/sp_15"
-                />
-        </RelativeLayout>
-        <android.support.v7.widget.RecyclerView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            app:layoutManager="@{LayoutManagers.linear(LinearLayoutManager.HORIZONTAL)}"
-            app:recyclerAdapter="@{viewModel.shopAdapter}"
-            />
 
-        <RelativeLayout
+            <android.support.design.widget.CollapsingToolbarLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@color/color_white"
+                app:contentScrim="?attr/colorPrimary"
+                app:layout_scrollFlags="scroll|enterAlwaysCollapsed"
+                >
+                <RelativeLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    >
+                    <LinearLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:orientation="vertical"
+                        >
+                        <RelativeLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:background="@color/color_white"
+                            >
+
+                            <TextView
+                                android:layout_width="@dimen/dp_250"
+                                android:layout_height="wrap_content"
+                                android:layout_marginEnd="@dimen/dp_30"
+                                android:layout_marginRight="@dimen/dp_30"
+                                android:drawableLeft="@drawable/ic_store_24dp"
+                                android:drawablePadding="@dimen/dp_5"
+                                android:drawableStart="@drawable/ic_store_24dp"
+                                android:fontFamily="@string/font_serif"
+                                android:gravity="center_vertical"
+                                android:padding="@dimen/dp_5"
+                                android:text="@string/top_shop"
+                                android:textColor="@color/color_orange_mos"
+                                android:textSize="@dimen/sp_17"
+                                android:textStyle="bold"
+                                tools:ignore="UnusedAttribute"
+                                />
+                            <ImageButton
+                                android:layout_width="@dimen/dp_35"
+                                android:layout_height="@dimen/dp_35"
+                                android:layout_alignParentEnd="true"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:layout_alignParentRight="true"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.